frontend/src/components/Issue/IssueSidebar.vue

```diff
@@ -31,6 +31,8 @@
           :disabled="!allowEditAssignee"
           :selectedId="create ? (issue as IssueCreate).assigneeId : (issue as Issue).assignee?.id"
           :allowed-role-list="['OWNER', 'DBA']"
+          :project="project"
+          :show-project-owner="allowProjectOwnerAsAssignee"
```

Comment:
I'm not quite familiar with these so please help me confirm that the available assignees are project owners if the assignee group is set to "project owners". 

The assignee group policy value should be conceived as a final value which overwrites the default value (a small modification of original design). 
So if no value is set, we use the default and the available assignee is workspace owners and DBAs.
If we set it to "PROJECT_OWNER", the available assignee is the project owners.
By the way, we may also add "WORKSPACE_OWNER_DBA_PROJECT_OWNER" in the future, and the available assignee is the workspace owners + DBAs + project owners.

Reply:
I see. It might be a misunderstanding of mine. I used to think that "PROJECT_OWNER" means both. I will change it.

Reply:
You got it right, it's actually a change of the original design.

Reply:
Updated